tidb-cloud/use-chat2query-api.md

```diff
@@ -251,10 +251,11 @@ curl --digest --user ${PUBLIC_KEY}:${PRIVATE_KEY} --request POST 'https://<regio
 }'
 ```
 
-In the preceding code, the request body is a JSON object with the following properties:
+The request body is a JSON object with the following properties:
 
 - `cluster_id`: _string_. A unique identifier of the TiDB cluster.
 - `database`: _string_. The name of the database.
+- `data_summary_id`: _integer_. The ID of the data summary used to generate SQL. This property only takes effect if `cluster_id` and `database` are not provided. If you do not specify `cluster_id`, `database`, and this property, the API uses the default data summary.
```

Comment:
What's the default data summary? Is it generated by Data Service for all databases automatically?